### DISCUSSION TITLE
FontStyle.Italic seems to be ignored for a font loaded via FontCollection?

### DISCUSSION THREAD

**Discussion by QuantumToasted** [SixLabors/Fonts#177] [Q&A] [answered]
Hello. I'm new to loading fonts from files versus using the `SystemFonts.CreateFont()` method. I successfully loaded a .ttf font file via  `FontCollection#Install(...)`, but I can't seem to apply things like italics to the `FontFamily` that `Install()` returns.

Here's a small reproduction:
```cs
using var canvas = new Image<Rgba32>(300, 100, Rgba32.ParseHex("FFFFFF"));
var collection = new FontCollection();
var fontFamily = collection.Install("Data/Fonts/TF2secondary.ttf");
var regularFont = fontFamily.CreateFont(20, FontStyle.Regular);
var italicFont = fontFamily.CreateFont(20, FontStyle.Italic);
canvas.Mutate(x =>
{
    x.DrawText("This text should be regular.", regularFont, Color.Black, new PointF(10, 10));
    x.DrawText("This text should be italics.", italicFont, Color.Black, new PointF(10, 50));
});
await using var output = new MemoryStream();
await canvas.SaveAsPngAsync(output);
```
And the outputted image: 
![image](https://user-images.githubusercontent.com/30324210/123910861-a8ea6500-d940-11eb-91b3-f9b4bd3faf4f.png)

I am able to use italics with this font in something like Word: 
![image](https://user-images.githubusercontent.com/30324210/123910949-c28bac80-d940-11eb-8935-7a3503424b41.png)

I'm not sure if this is a limitation of the library, or if Word or other programs which use italics for fonts do some hackery i'm not aware of, or if I'm simply making assumptions that my code is correct when I'm actually missing something or messing something up. Any advice or solutions would be appreciated.

The font is [TF2 Secondary](https://www.dafont.com/tf2-secondary.font) found on dafont for usage in testing your own repro.

**Edit: It seems from your response in [this discussion](https://github.com/SixLabors/Fonts/discussions/169#discussioncomment-626588) that maybe it has to do with the fact I'm just loading a single font file, and not a supposed "TF2 Secondary Italic" font file? As far as I can tell one doesn't exist, or as least I'm only installing a single .ttf file, so now I'm leaning towards "Word and other programs are doing some hackery to force the font to be italic" being the cause.*

**Comment by JimBobSquarePants** [comment, marked as answer]
>so now I'm leaning towards "Word and other programs are doing some hackery to force the font to be italic" being the cause.

Yep. That's exactly what they do. https://graphicdesign.stackexchange.com/a/42413

You could do something similar by passing a `Matrix3x2` via the `DrawingOptions` to the `DrawText` method. Use the `AffineTransformBuilder` to create the skew matrix using `TextMeasurer` to provide the text bounds to generate the transform.
